xpublish-exploring/lme.py

```diff
@@ -53,14 +59,13 @@ def get_region_dataset(dataset_id: str, region_id: str):
 
             # lat_slice = slice(bbox[2], bbox[3])
             lat_slice = slice(bbox[2], bbox[3])  # air_temperature lats are descending
-            # lon_slice = slice(bbox[0], bbox[1])
-            
+            lon_slice = slice(bbox[0], bbox[1])
 
             # print(lat_slice, lon_slice)
 
             dataset = deps.dataset(dataset_id)
 
-            sliced = dataset.cf.sel(latitude=lat_slice)
+            sliced = dataset.cf.sel(latitude=lat_slice, longitude=lon_slice)
```

Comment:
Does http://localhost:9000/datasets/air_temperature/lme/EC/air/mean return valid data?

Reply:
No, I couldn't get it to work. If you can provide a working URL i.e. overlapping dataset / bbox that would be great. Otherwise it can be part of the course as discussed.